fuel-core/src/service/modules.rs

```diff
@@ -301,3 +305,31 @@ impl BlockProducerRelayer for MaybeRelayerAdapter {
             .unwrap_or_default())
     }
 }
+
+struct PoACoordinatorAdapter {
```

Comment:
Could we move this to an adapters module, e.g. similar to this? https://github.com/FuelLabs/fuel-core/blob/master/docs/architecture.md#adapters-fuel_coreserviceadapterspoa_consensus